libphx/src/shader_var_type.rs

```diff
@@ -0,0 +1,96 @@
+use crate::common::*;
+use crate::math::Vec3;
+use crate::math::Vec4;
+use crate::math::{IVec2, IVec3, IVec4, Vec2};
+use crate::matrix::*;
+use crate::tex1d::*;
+use crate::tex2d::*;
+use crate::tex3d::*;
+use crate::tex_cube::*;
+use crate::*;
+
+pub type ShaderVarType = i32;
+
+const SHADER_VAR_TYPE_GLSL_NAMES: [&str; 13] = [
+    "float",
```

Comment:
Could we make this an array of structs with the name, glsl name, and size, then change `shader_var_type_get_glsl_name(...)` to `shader_var_type_get(index).glsl_name` in `ShaderVarType_GetGLSLName`, but we can also simplify `ShaderVarType_GetName` and `ShaderVarType_GetSize`.

Reply:
Done